crates/astria-core/CHANGELOG.md

```diff
@@ -31,6 +31,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   sequencers to utilize CometBFT's extended votes for getting consensus on price
   feed data [#2085](https://github.com/astriaorg/astria/pull/2085).
 - Add `name` field to domain `ValidatorUpdate` action [#2089](https://github.com/astriaorg/astria/pull/2089).
+- Add `CheckedTransaction`, `CheckedAction` and `Checked...` wrappers for all
+  action types [#2142](https://github.com/astriaorg/astria/pull/2142).
+- Add new ABCI error code 18: `TRANSACTION_FAILED_CHECK_TX` [#2142](https://github.com/astriaorg/astria/pull/2142).
```

Comment:
This doesn't seem to be true: 1. it already exists in main, and 2. it's seems to be error code 18?

Reply:
Fixed in [2270e90](https://github.com/astriaorg/astria/pull/2142/commits/2270e90cc6f02a75e4ce0ea377e084f7031cace3) (this was added in #2133 along with three other codes, all added to the changelog now).